src/views/TradingCompetition/components/ShareImageModal.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useRef, useState, useEffect } from 'react'
+import { Modal, Flex, Button, Text, Skeleton, Box } from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+import useI18n from 'hooks/useI18n'
+import FlippersShare from '../pngs/flippers-share.png'
+import StormShare from '../pngs/storm-share.png'
+import CakersShare from '../pngs/cakers-share.png'
+import ProfileMask from '../pngs/share-profile-mask.png'
+import MedalGold from '../pngs/medals/medal-gold.png'
```

Comment:
We already have these as .tsx / wrapped SVGs but canvas is dumb and it either wants a hacky way to get raw SVGs in, or images. Either way I'd need to re-add these assets. So... I've re-added these images (each one sub 10kb).